index.html

```diff
@@ -6,6 +6,392 @@
 	<link rel="stylesheet" type="text/css" href="index.css">
 </head>
 <body>
-	
+
+	<!-- intro -->
+	<header class="row">
+
+		<img src="assets/yellow-pill.svg" alt="yellow pill">
+		<img src="assets/more-pills.svg" alt="a blue pill, a purple pill, and a yellow pill">
+		
+		<span class="subTitle">A Viget Exploration</span>
```

Comment:
Hmmm. Perhaps you don't need the class at all anymore since it is inside the ```<h1>```. You could instead target any ```<span>```  inside an ```<h1>``` now using ```h1 span { property: value; }``` inside of your CSS.